src/ralph/dns/dnsaas.py

```diff
@@ -47,7 +47,7 @@ def get_dns_records(self, ipaddresses):
         ipaddresses = [('ip', i) for i in ipaddresses]
         url = urljoin(
             settings.DNSAAS_URL,
-            'api/records/?{}'.format(
+            'api/v2/records/?{}'.format(
```

Comment:
or create some function to build url - some like this:
``
def build_url(resource_name, id=None, version='v2', *get_args)
``